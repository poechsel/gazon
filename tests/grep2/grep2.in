login u1
pass p1
grep @epfl.ch
