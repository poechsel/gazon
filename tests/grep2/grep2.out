dir1/dir2/F2.txt
dir1/F2.txt
